service/history/timerQueueAckMgr.go

```diff
@@ -118,17 +118,17 @@ func (t timerKeys) Swap(i, j int) {
 
 // Less implements sort.Interface
 func (t timerKeys) Less(i, j int) bool {
-	return compareTimerIDLess(&t[i], &t[j])
+	return CompareTimerIDLess(&t[i], &t[j])
 }
 
-func newTimerKey(time time.Time, taskID int64) *timerKey {
+func NewTimerKey(time time.Time, taskID int64) *timerKey {
```

Comment:
Shall we deprecate the TimerKey and use `tasks.Key` instead?

Reply:
Yes. Good to know. I will update this.